gateway-ha/src/test/java/com/lyft/data/gateway/ha/router/TestQueryHistoryManager.java

```diff
@@ -42,8 +41,8 @@ public void testSubmitAndFetchQueryHistory() {
     queryDetail.setUser("test@ea.com");
     queryDetail.setQueryText("select 1");
     for (int i = 0; i < 2; i++) {
-      queryDetail.setQueryId(String.valueOf(Time.absoluteTime()));
-      queryDetail.setCaptureTime(Time.absoluteTime());
+      queryDetail.setQueryId(String.valueOf(new Timestamp(System.currentTimeMillis()).getTime()));
```

Comment:
`timestamp.getTime()` returns same value as `System.currentTimeMillis()`

Reply:
`String.valueOf(new Timestamp(System.currentTimeMillis()).getTime())` can be replaced as `String.valueOf(System.currentTimeMillis())`